src/templates/container-hotkey.jsx

```diff
@@ -1,53 +1,53 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
```

Comment:
Yeah thanks @copons. I think it will be easy enough to automate that change later on. Should I be updating it now? I don't know.